ml/backend/ggml/ggml/src/ggml.go

```diff
@@ -90,4 +105,37 @@ var OnceLoad = sync.OnceFunc(func() {
 			visited[abspath] = struct{}{}
 		}
 	}
+
+	var attrs []slog.Attr
```

Comment:
Should we split this aggregation to its own function which is called from `OnceLoad`? Seems like a lot of logic to stick in the variable, loses a bit of its context.

Reply:
Not really. This is relevant to the loading. It outputs the system information once it's loaded once. I might consider is to split this into a struct that implements `slog.LogValue` to defer evaluation though